spec/unit/blockscore/collection/member_spec.rb

```diff
@@ -0,0 +1,8 @@
+module BlockScore
+  class Collection
+    RSpec.describe Member do
+      describe '.save' do
+      end
+    end
```

Comment:
We currently do not use this feature, so I have no exposed way of testing it. This is here as a reminder to test it, when we extend from it.